### DISCUSSION TITLE
Sponsors

### DISCUSSION THREAD

**Discussion by Thanaen** [corvudev/corvu#22] [General]
Do you plan to accept sponsors for this project?
I have a feeling that it's going to become the Radix UI of SolidJS, and that would open the door to a lot of developers in my opinion. So I think it would be interesting to support the project and, in a way, pay for the time you spend on it!

Thanks a lot!

**Comment by GiyoMoon** [comment]
Hey Thanaen!
Thanks a lot for this very kind message :) I am planning to accept sponsors for this project yes! I've already set up my GitHub sponsors   (https://github.com/sponsors/giyomoon) page a few months ago but was hesitant to add it to corvu. I'll do it in the near future.
